react/features/base/react/components/native/HeaderWithNavigation.js

```diff
@@ -48,17 +53,18 @@ class HeaderWithNavigation extends Component<Props> {
      * @inheritdoc
      */
     render() {
-        const { onPressBack, onPressForward } = this.props;
+        const { hideHeaderWithNavigation, onPressBack, onPressForward } = this.props;
 
         return (
-            <Header>
-                { onPressBack && <BackButton onPress = { onPressBack } /> }
-                <HeaderLabel labelKey = { this.props.headerLabelKey } />
-                { onPressForward && <ForwardButton
-                    disabled = { this.props.forwardDisabled }
-                    labelKey = { this.props.forwardLabelKey }
-                    onPress = { onPressForward } /> }
-            </Header>
+            !{ hideHeaderWithNavigation }
```

Comment:
> This looks pretty good (functionality wise). A few remarks:
> 
> 1. If a participant quits while the the participants pane & drawer are open for it, there is an error:
> 
> <img alt="Screenshot 2021-07-02 at 12 11 52" width="457" src="https://user-images.githubusercontent.com/37841821/124251621-d137b680-db2e-11eb-967e-dda481fef3b2.png">
> 
> 1. There are a few design inconsistency:
> 
> * The upper right close button looks way to big. Please check the dimensions from the design and use the tokens.
> * The same applies for the bottom buttons (Mute and ... button)
> * The avatar  from the participant list and the icons from  actions menu looks like are not middle aligned.
> * There are a few keys missing: `participantsPane.actions.mute` & `participantsPane.actions.muteEveryoneElse`
> * The knob for the slider is way too big.
> 
> 1. Can we get rid of the warning regarding the Slider? It looks like it's deprecated and we have to use the one from react-native-community instead of the core one.

Everything is fixed except the knob. For the Slider component we have no thumbSize prop. 
There is an issue on the community that is still open
https://github.com/callstack/react-native-slider/issues/97.